scala-package/README.md

```diff
@@ -70,123 +122,24 @@ make SCALA_TEST_ARGS=-Dsuites=org.apache.mxnet.NDArraySuite scalaunittest
 If everything goes well, you will find jars for `assembly`, `core` and `example` modules.
 Also it produces the native library in `native/{your-architecture}/target`, which you can use to cooperate with the `core` module.
 
-Once you've downloaded and unpacked MNIST dataset to `./data/`, run the training example by
-
-```bash
-java -Xmx4G -cp \
-  scala-package/assembly/{your-architecture}/target/*:scala-package/examples/target/*:scala-package/examples/target/classes/lib/* \
-  org.apache.mxnet.examples.imclassification.TrainMnist \
-  --data-dir=./data/ \
-  --num-epochs=10 \
-  --network=mlp \
-  --cpus=0,1,2,3
-```
-
-If you've compiled with `USE_DIST_KVSTORE` enabled, the python tools in `mxnet/tracker` can be used to launch distributed training.
-The following command runs the above example using 2 worker nodes (and 2 server nodes) in local. Refer to [Distributed Training](http://mxnet.incubator.apache.org/how_to/multi_devices.html) for more details.
-
-```bash
-tracker/dmlc_local.py -n 2 -s 2 \
-  java -Xmx4G -cp \
-  scala-package/assembly/{your-architecture}/target/*:scala-package/examples/target/*:scala-package/examples/target/classes/lib/* \
-  org.apache.mxnet.examples.imclassification.TrainMnist \
-  --data-dir=./data/ \
-  --num-epochs=10 \
-  --network=mlp \
-  --cpus=0 \
-  --kv-store=dist_sync
-```
-
-Change the arguments and have fun!
+Examples & Usage
```

Comment:
we shouldn't remove the example for training, can you please update the example?

Reply:
@nswamy I removed the example snippet to run here, as it seemed a bit out of place right under the section on build from source. 
I can however, direct the users to this script ```run_train_mnist.sh``` in order to run the training script for MNIST. 
Will that work ?

Reply:
Discussed offline, there would be a separate README for training code, elsewhere, and the main scala readme would point to it from here.

Reply:
I have added pointers to the training API usage in this commit : https://github.com/apache/incubator-mxnet/pull/13582/commits/91392c4d35a1f3781182ccab4d3ddd389647b9bf 

The improvement in usage of training APIs available on the website here : http://mxnet.incubator.apache.org/api/scala/model.html should be a part of separate task. 

Have added a JIRA item for it  : https://issues.apache.org/jira/browse/MXNET-1265